tamr_unify_client/models/attribute/type.py

```diff
@@ -0,0 +1,45 @@
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class AttributeType(BaseResource):
+
+    @classmethod
+    def from_json(cls, client, data, api_path):
+        # Need to fill in this path
+        # This attribute might be part of the collection on a dataset, or
+        # this attribute might be part of the collection on an attribute
+        return super().from_data(client, data, api_path)
+
+    @property
+    def relative_id(self):
+        return self.api_path
+
+    @property
+    def base_type(self):
+        """:type: str"""
+        return self._data.get("baseType")
+
+    @property
+    def inner_type(self):
+        """:type: :class:`~tamr_unify_client.models.attribute.type.AttributeType`"""
+        if "innerType" in self._data:
+            alias = self.api_path + "/type"
```

Comment:
Author judgment: inserting `type` here gives a distinct URI to the type, but ends up with URIs like:
`datasets/1/attributes/id/geom/type/attributes/point/type`
I believe this is correct.